GitExtensions/Program.cs

```diff
@@ -165,13 +169,16 @@ private static void RunApplication()
                 // Avoid replacing the ExitCode eventually set while parsing arguments,
                 // i.e. assume -1 and afterwards, only set it to 0 if no error is indicated.
                 Environment.ExitCode = -1;
-                if (commands.RunCommand(args))
+                if (commands.RunCommand(args, out saveAppSettings))
```

Comment:
So the only way to have `saveAppSettings=true` here is if `args.Contains("browse")` right? 
Why not make this decision here rather than complicate the code in layers below?